cobrand_hackney/static/js.js

```diff
@@ -65,12 +65,26 @@ function show_hide(input, value, ids) {
     }
 }
 
+function show_hide_checkbox(input, show_when_checked, ids) {
+    var input = document.querySelector(`input[type='checkbox'][name='${input}']`);
+    if (!input) {
+	return;
+    }
+    let f = function() {
+	let show = input.checked && show_when_checked || !input.checked && !show_when_checked;
```

Comment:
Perhaps some brackets in this just to be clear what it's doing? I assume it works fine